plugins/woocommerce/includes/admin/wc-admin-functions.php

```diff
@@ -508,27 +508,29 @@ function wc_render_invalid_variation_notice( $product_object ) {
 
 	// Check if a variation exists without pricing data.
 	// phpcs:disable WordPress.DB.PreparedSQL.NotPrepared
-	$invalid_variation_count = $wpdb->get_var(
+	$valid_variation_count = $wpdb->get_var(
```

Comment:
Thanks for fixing this up... it was so confusing that this was incorrectly named `$invalid_variation_count` previously! Most of my time reviewing this was just making sure that I wasn't missing something with this obvious of a misname! 😄